sql/ksp/_ksp.sql

```diff
@@ -28,10 +31,57 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 ---------------
 
 --v2.6
+-- CREATE FUNCTION _pgr_ksp(
```

Comment:
If you are marking a function for deletion, in SQL use `/*   */` to wrap the unwanted code
and a  `-- TODO if 0`  <--- makes it easier to find the unwanted code that was forgotten to remove

Reply:
Forgot to mention, you can not delete an SQL function, this is a 3.6.0 which can only add functions from 3.5, you will have problems when testing updates. 
TRSP had a similar problem and the way around was: 
https://github.com/pgRouting/GSoC-pgRouting/blob/81bcd0c65fcd127b9497cda5375e0800b7ec7ecc/sql/trsp/_trsp.sql#L59
As you can see the old function remains, but the new one to be kept on v4 is named differently.

Reply:
okay, Got it Thank you.

Reply:
> Forgot to mention, you can not delete an SQL function, this is a 3.6.0 which can only add functions from 3.5, you will have problems when testing updates. TRSP had a similar problem and the way around was:
> 
> https://github.com/pgRouting/GSoC-pgRouting/blob/81bcd0c65fcd127b9497cda5375e0800b7ec7ecc/sql/trsp/_trsp.sql#L59
> 
> 
> As you can see the old function remains, but the new one to be kept on v4 is named differently.

Can I change SQL function?